pkg/api/v1/client/direct.go

```diff
@@ -85,11 +91,17 @@ func (c DirectScriptsAPI) GetExecution(scriptID, executionID string) (execution
 
 // ListExecutions list all executions for given script name
 func (c DirectScriptsAPI) ListExecutions(scriptID string, limit int, tags []string) (executions testkube.ExecutionsResult, err error) {
-	var uri string
+
+	uri := "/executions"
+
+	if scriptID != "" {
+		uri = fmt.Sprintf("/scripts/%s/executions", scriptID)
```

Comment:
remove this one handle only executions by ID

Reply:
@exu Do we want to change list executions command?

Reply:
no sorry I've missed this one - it's ok